device_backends/DummyBackend/src/DummyBackend.cc

```diff
@@ -49,6 +49,7 @@ namespace ChimeraTK {
     TRY_REGISTER_ACCESS(_barContents[bar].at(address / sizeof(int32_t)) = data;);
   }
 
+/*	
```

Comment:
Remove commented code which has been replaced with new code.

Reply:
This is Jan Timm who commented this part of code. I was not looking into DummyBackend.h/cc files. I will remove it, especially that it looks that commented lines are exact copy of code below (from lines 64-79)